docs/man/dune.1.md

```diff
@@ -0,0 +1,285 @@
+    If optional arguments are provided, it creates specific CURRENCY (default "SYS") 
+    with maximum amount of MAX_VALUE and initial value of INITIAL_VALUE.
+
+**`--send-action <ACCOUNT> <ACTION> <DATA> <PERMISSION>`**
+
+    Send an action to a specified account with given data and permission.
+
+**`--get-table <ACCOUNT> <SCOPE> <TABLE>`**
+
+    Print data from a given table.
+
+**`--activate-feature <CODENAME>`**
+
+    Activate a given protocol feature.
+
+**`--list-features`**
+
+    Print a list of available protocol features.
+
+**`--version`**
+
+    Display the current version of DUNE.
+
+**`--version-all`**
+
+    Display the current versions of DUNE, CDT, and leap.
+
+**`--debug`**
+
+    Print additional information useful for debugging, such as running docker commands.
+
+**`--upgrade`**
+
+    Upgrade DUNE image to the latest version.
+
+**`--leap [LEAP_VERSION]`**
+
+    Set the version of leap. If no version is provided, display available leap versions.
+
+**`--cdt [CDT_VERSION]`**
+
+    Set the version of CDT (Contract Development Toolkit). If no version is provided, display available CDT versions.
+
+**`--create-project <PROJ_NAME> <DIR>`**
+
+    Create a smart contract project at the specified location.
+
+**`--add-app <PROJ_DIR> <APP_NAME> <LANG> [CMPLR_OPTS] [LINK_OPTS]`**
+
+    Add an application to the specified smart contract project.
+
+**`--add-lib <PROJ_DIR> <LIB_NAME> <LANG> [CMPLR_OPTS] [LINK_OPTS]`**
+
+    Add a library to the specified smart contract project.
+
+**`--add-dep <PROJ_DIR> <OBJ_NAME> <DEP_NAME> [LOCATION] [TAG/RELEASE] [HASH]`**
+
+    Add a dependency to the specified smart contract project.
+
+**`--remove-app <PROJ_DIR> <APP_NAME>`**
+
+    Remove an application from the specified smart contract project.
+
+**`--remove-lib <PROJ_DIR> <LIB_NAME>`**
+
+    Remove a library from the specified smart contract project.
+
+**`--remove-dep <PROJ_DIR> <OBJ_NAME> <DEP_NAME>`**
+
+    Remove a dependency from the specified smart contract project.
+
+**`--update-app <PROJ_DIR> <APP_NAME> <LANG> [CMPLR_OPTS] [LINK_OPTS]`**
+
+    Update an application in the given smart contract project.
+
+**`--update-lib <PROJ_DIR> <LIB_NAME> <LANG> [CMPLR_OPTS] [LINK_OPTS]`**
+ 
+    Update a library in the given smart contract project.
+
+**`--update-dep <PROJ_DIR> <OBJ_NAME> <DEP_NAME> [LOCATION] [TAG/RELEASE] [HASH]`**
+
+    Update a dependency in the given smart contract project.
+ 
+**`--build-project <PROJ_DIR>`**
+
+    Build the given smart contract project.
+                        
+**`--clean-build-project <PROJ_DIR>`**
+
+    Clean the given project and rebuild it from scratch.
+                        
+**`--validate <PROJ_DIR>`**
+
+    Validate the given smart contract project.
+                        
+**`--populate <PROJ_DIR>`**
+
+    Populate the given smart contract project.
+
+# BUGS
+
+Please submit bug reports online at https://github.com/AntelopeIO/DUNE/issues
+
+# SEE ALSO
```

Comment:
```suggestion
# SEE ALSO

  **cdt**(1), **leap**(1)
  
#  COLOPHON
```

I think this might be better?

Reply:
Or something like this? "See also" would normally describe other man pages. I'm not 100% sure here if we should have the hyperlink here or in "colophone"? Maybe @iamveritas has an opinion?